composer.json

```diff
@@ -17,12 +17,12 @@
         }
     ],
     "require": {
-        "php": ">=7.1"
+        "php": ">=7.2"
```

Comment:
Because of this change, travis configuration needs to be updated

Reply:
I see php 7.2 is configured in .travis.yml on 2.0 branch

Reply:
Yes, but you need to exclude 7.1